projects/addon-charts/components/line-chart/line-chart.component.ts

```diff
@@ -96,7 +96,9 @@ export class TuiLineChartComponent {
         return this.hover$.pipe(distinctUntilChanged(), tuiZoneOptimized(this.zone));
     }
 
-    protected get hintContent(): PolymorpheusContent<TuiLineChartHintContext<TuiPoint>> {
+    protected get hintContent(): PolymorpheusContent<
+        TuiLineChartHintContext<readonly TuiPoint[]>
```

Comment:
If the context type has changed, probably we need to update some examples that relied on the old variant